iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/PhaseTapChangerImpl.java

```diff
@@ -121,7 +121,7 @@ public void allocateVariantArrayElement(int[] indexes, final int sourceIndex) {
     }
 
     @Override
-    protected String getTapChangerAttribute() {
-        return "phaseTapChanger";
+    protected Supplier<String> getTapChangerAttribute() {
+        return () -> "phase" + parent.getTapChangerAttribute();
```

Comment:
It's hard to say but is it necessary to return a supplier here, specially if this method is always called from a supplier. Suppliers have been added to avoid string creation by concatenation, but with this solution we have two levels of `Supplier`s.

From my point of view, it's equivalent to keep a `String` here.

Reply:
I added the `supplier` to follow the same optimization that was made when building the string for the `notifyUpdate` methods in the `NetworkListener`. 

I agree that if `getTapChangerAttribute` is called only in the context of `notifyUpdate` methods that already contain the `Supplier` it is redundant.

If there is no other use for this method, I agree on keeping the return value as a string.

Please confirm and I will make the change.

Reply:
I’m pretty sure that these methods are used only in that context. So I suggest to change the return type to String.
If we encounter any performance issue, we’ll fix it.

Reply:
Agree. I will make the change